doc/explanation/clustering.md

```diff
@@ -4,27 +4,39 @@
 ```{youtube} https://www.youtube.com/watch?v=nrOR6yaO_MY
 ```
 
-LXD can be run in clustering mode, where any number of LXD servers share the same distributed database and can be managed uniformly using the `lxc` client or the REST API.
+To spread the total workload over several servers, LXD can be run in clustering mode.
+In this scenario, any number of LXD servers share the same distributed database that holds the instances and the general configuration.
```

Comment:
To avoid implying that the database stores the instances (as opposed to just their configuration) might it be clear to say here:

"In this scenario, any number of LXD servers share the same distributed database that holds the configuration for the cluster members and their instances."?